### DISCUSSION TITLE
ptuning训练遇到的情况，几个问题想请教一下

### DISCUSSION THREAD

**Discussion by youyouge** [zai-org/ChatGLM3#603] [Q&A]
第一：为什么训练出来的模型是问答式的，是因为数据量太少吗，格式和官方给的测试数据集格式一样都是类型#描述类型#描述，训练使用的是./scripts/finetune_pt.sh 参数只改了sourse和target还有step三个数值。
第二：为什么训练完的模型，使用“cd ../composite_demo
MODEL_PATH="THUDM/chatglm3-6b" PT_PATH="path to p-tuning checkpoint" streamlit run main.py”启动UI界面,使用API启动，最终运行的还是原来的chatglm3-6b，不是微调后的模型
第三：数据集的格式“类型#描述类型#描述”，后面只描述一个或多个描述词或者类型词，可以得到回复吗，是经过ChatGLM3-6B思考后的回复还是数据集的内容完全一模一样的回复还是会参考数据集答案的格式来回复。。。。

**Comment by 123sghn** [comment]
我也是第二条遇到了问题，如果采用python inference.py \
    --pt-checkpoint "path to p-tuning checkpoint" \
    --model THUDM/chatglm3-6b 这个倒是也可以，但这个是针对输入输出格式的，所以你问啥它基本啥也回答不了。多轮对话格式的按照官网文档的部署，微调后一点效果都没有。

**Comment by 123sghn** [comment]
大佬，请问这个问题你解决了？

**Reply by youyouge** [reply]
大佬，你解决了没有。。。

**Reply by 123sghn** [reply]
没有，最近在忙别的事情，也没有大佬回复我们的问题。（ku）

**Reply by 123sghn** [reply]
我现在觉得可能是我们的数据集太小了，如果使用官方的数据集效果应该可以，等我期末周过后试一下

**Comment by zRzRzRzRzRzRzR** [comment]
move to #253
